src/main/java/net/greatstart/validators/NewUserValidator.java

```diff
@@ -0,0 +1,23 @@
+package net.greatstart.validators;
+
+import net.greatstart.dto.DtoUser;
+import net.greatstart.services.UserService;
+import org.springframework.validation.Errors;
+
+public class NewUserValidator {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)